gridlab-d/gridlab-d
# Feature/1202 - three-phase house updates

#### What's this Pull Request do?
This pull request updates the three-phase house to properly connect in "standard GridLAB-D ways" to general node devices (node, meter, load), and allows more triplex connections (triplex_node, triplex_load, triplex_meter).

The pull request "normalizes" load and triplex_load to be more like normal nodes, instead of semi-special implementations.  This should help with future implementations that go this way.

Updated the test_commercial_meters autotest to be more explicit, as well as replicated to FBS and NR.

Also some general fixes to "measured_power" calculations in load and triplex_load (they had typos).

As of January 9, 2020, this should be up to date with feature/1173, and should be an easy merge.

#### Where should the reviewer start?
Pull down feature/1202 and make sure the three-phase house connection behaves as expected.  The new test_commercial_meters_FBS.glm and test_commercial_meters_NR.glm have explicit ZIP loads to test ZIP contributions and how they get applied to powerflow.

#### How should this be tested?
Run a three-phase house and look at the results.

Alternatively, verify the updated test_commercial_meters_XXX.glm files are correct, and that all autotests still pass.

#### Any background context you want to provide?
No

#### What are the relevant issues?
#1185 and #1202 

#### Questions:
No
Yes, but it was already there.  Look at the build logs.
